### DISCUSSION TITLE
Support Reactive ignore attribute on ~all nodes

### DISCUSSION THREAD

**Discussion by totalhack** [formkit/formkit#707] [Ideas]
I'm working with a multi-step form implementation that dynamically hides steps. When it hides steps and they are never visited, they still try to get validated on submit. Previously I was using an if condition to handle this but with version 0.17 that no longer works (breaks other dynamic step behavior). I'd like the ability to dynamically add an ignore flag to my sections so the submit handler ignores steps that are not actively part of the flow.

**Comment by pjrusso** [comment]
Hi - after seeing your reply to [https://discord.com/channels/793529058377072650/940340536747896842/1109136922745520270](url) @andrew-boyd it reminded me a feature that I have seen in another (bespoke) implementation - that is the ability to suspend any 'required' validation rules so a partially complete form could be saved (i.e. missing data is ok, but not anything else that fails validation).  I have a play with your example Andrew - many thanks for the inspiration!
